### DISCUSSION TITLE
The unit size for VirtualMachine.disk and VirtualDisk.size changed starting from NetBox 4.1.0

### DISCUSSION THREAD

**Discussion by lukedaemon** [netbox-community/netbox#17484] [General]
Installed version: **4.1.1**

NetBox version **4.1.0** introduced this change: "The unit size for VirtualMachine.disk and VirtualDisk.size has been changed from 1 gigabyte to 1 megabyte. Existing values will be adjusted automatically during the upgrade process."

Now during the addition or modification of a virtual disk, the size is expressed in MB, while it is displayed in GB. However, I noticed NetBox performs the conversion by dividing by 1000 instead of 1024, which results in a discrepancy between the two units of measurement, as 1 GB equals 1024 MB, not 1000. Isn't it ?

![image](https://github.com/user-attachments/assets/7c561c8e-048d-4b28-a64a-c79e87b4a589)

**Comment by jeremystretch** [comment]
[A gigabyte is one billion bytes.](https://en.wikipedia.org/wiki/Gigabyte#Definition)

**Reply by lukedaemon** [reply]
Hello @jeremystretch ,
I completely agree with you and Wikipedia (GB is different by GiB).

However, due to historical usage and practical reasons related to computer operation, GB it is often used as a synonym for GiB on certain OS.

Bye,
Luca
